### ISSUE TITLE
Unable to open Realtek Audio Console

### ISSUE THREAD

**Issue by ssrubsd** [AlanFinotty/AAFDCHDriverMod#192]
Use environment: Windows11
The Realtek Audio Console cannot be opened normally, and the window interface just crashes without any errors, but it can be opened normally in a few cases.
What is the reason for this and how can I fix it?

**Comment by ketxxx** [comment]
Exactly the same happening here on Win10 Pro x64 even when installing just the driver, Realtek CP and ASIO.

**Comment by ketxxx** [comment]
I've looked into this problem @ssrubsd and you can resolve it by downloading the .RAR archive version of the driver, browsing to Others > Apps > Realtek and deleting "RealtekAudioControl". Now download the 9298.1 .RAR archive, browse to the same location and extract the "RealtekAudioControl" package (drag and drop). Its a slightly older version of the Realtek CP that works perfectly fine.

**Comment by ssrubsd** [comment]
> I've looked into this problem @ssrubsd and you can resolve it by downloading the .RAR archive version of the driver, browsing to Others > Apps > Realtek and deleting "RealtekAudioControl". Now download the 9298.1 .RAR archive, browse to the same location and extract the "RealtekAudioControl" package (drag and drop). Its a slightly older version of the Realtek CP that works perfectly fine.

ok ,i'll try, thanks! : )

**Comment by ketxxx** [comment]
np ;) The problem is twofold, for some reason Windows is blocking the package flagging it as coming from another computer and even when you unblock the package the control panel seems to be bugged anyway it has an error about not being able to connect to the realtek service.

**Comment by ssrubsd** [comment]
> np ;) The problem is twofold, for some reason Windows is blocking the package flagging it as coming from another computer and even when you unblock the package the control panel seems to be bugged anyway it has an error about not being able to connect to the realtek service.

Yes, I also found that replacing with the old version worked at first, then the problem persisted. Hope Alan can fix this soon
